bin/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+      "node/no-unpublished-require": 0
+  }
+}
```

Comment:
We can remove the last empty line.

Reply:
> We can remove the last empty line.

there is no empty last line. The red circle with a red dash in it signifies no newline at the end of the file. You can verify this on disk.